content/en/docs/instrumentation/erlang/sampling.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Sampling
+weight: 37
+---
+
+## Sampling
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of
+traces that are generated by a system. The Erlang SDK offers several
+[head samplers](/docs/concepts/sampling#head-sampling).
+
+### Default behavior
+
+By default, all spans are sampled, and thus, 100% of traces are sampled. If you
+do not need to manage data volume, don't bother setting a sampler.
+
+### ParentBasedSampler
+
+When sampling, the `ParentBasedSampler` is most often used for
+[head sampler](/docs/concepts/sampling/#head-sampling). It uses the sampling
+decision of the Span's parent, or the fact that there is no parent, to know
+which secondary sampler to use.
+
+The sampler can be configured with environment variables `OTEL_TRACES_SAMPLER`
+and `OTEL_TRACES_SAMPLER_ARG` or using the Application configuration allows you
+to configure each of the 5 potential states of a Span's parent:
+
+- `root` - No parent
+- `remote_parent_sampled` - Parent is from a remote Span that is sampled
+- `remote_parent_not_sampled` - Parent is from a remote Span that is not sampled
+- `local_parent_sampled` - Parent is from a local Span that is sampled
+- `local_parent_not_sampled` - Parent is from a local Span that is not sampled
+
+#### TraceIdRatioBasedSampler
+
+Within the `ParentBasedSampler` the most common is the
+`TraceIdRatioBasedSampler`. It deterministically samples a percentage of traces
+that you pass in as a parameter.
+
+##### Environment Variables
+
+You can configure the `TraceIdRatioBasedSampler` with environment variables:
+
+```shell
+export OTEL_TRACES_SAMPLER="parentbased_traceidratio"
+export OTEL_TRACES_SAMPLER_ARG="0.1"
+```
+
+This tells the SDK to sample spans such that only 10% of Traces get created.
+
+##### Application configuration
+
+Example in the Application configuration with a root sampler for sampling 10% of
+Traces and using the parent decision in the other cases:
+
+<!-- prettier-ignore-start -->
+{{< tabpane langEqualsHeader=true >}}
+
+{{< tab Erlang >}}
+%% config/sys.config.src
+{sampler, {parent_based, #{root => {trace_id_ratio_based, 0.1},
+                           remote_parent_sampled => always_on,
+                           remote_parent_not_sampled => always_off,
+                           local_parent_sampled => always_on,
+                           local_parent_not_sampled => always_off}}}
+{{< /tab >}}
+
+{{< tab Elixir >}}
+# config/runtime.exs
+sampler: {:parent_based, %{root: {:trace_id_ratio_based, 0.1},
+                           remote_parent_sampled: :always_on,
+                           remote_parent_not_sampled: :always_off,
+                           local_parent_sampled: :always_on,
+                           local_parent_not_sampled: :always_off}}
```

Comment:
```suggestion
{sampler, {parent_based, #{root => {trace_id_ratio_based, 0.10},
                           remote_parent_sampled => always_on,
                           remote_parent_not_sampled => always_off,
                           local_parent_sampled => always_on,
                           local_parent_not_sampled => always_off}}}
{{< /tab >}}

{{< tab Elixir >}}
# config/runtime.exs
sampler: {:parent_based, %{root: {:trace_id_ratio_based, 0.10},
                           remote_parent_sampled: :always_on,
                           remote_parent_not_sampled: :always_off,
                           local_parent_sampled: :always_on,
                           local_parent_not_sampled: :always_off}}
```

Absolutely minor thing, but since you state "10%" in the text, then using `0.10` makes it easier to visually match the content than a `0.1` even if it's absolutely the same value in the end.